### PR TITLE
ci-operator/config/openshift/cincinnati: Opt into automated branching

### PR DESCRIPTION
**Pull request by wking**
Following [the docs][1].

[1]: https://docs.ci.openshift.org/docs/architecture/branching/#how-do-i-opt-my-repository-into-automated-branching

### CODE REVIEW

File: core-services/image-mirroring/openshift/mapping_origin_4_10 [hunk 1]
```diff
@@ -93,11 +93,13 @@ registry.ci.openshift.org/origin/4.10:csi-livenessprobe quay.io/openshift/origin
 registry.ci.openshift.org/origin/4.10:csi-node-driver-registrar quay.io/openshift/origin-csi-node-driver-registrar:4.10 quay.io/openshift/origin-csi-node-driver-registrar:4.10.0 quay.io/openshift/origin-csi-node-driver-registrar:latest
 registry.ci.openshift.org/origin/4.10:csi-snapshot-controller quay.io/openshift/origin-csi-snapshot-controller:4.10 quay.io/openshift/origin-csi-snapshot-controller:4.10.0 quay.io/openshift/origin-csi-snapshot-controller:latest
 registry.ci.openshift.org/origin/4.10:csi-snapshot-validation-webhook quay.io/openshift/origin-csi-snapshot-validation-webhook:4.10 quay.io/openshift/origin-csi-snapshot-validation-webhook:4.10.0 quay.io/openshift/origin-csi-snapshot-validation-webhook:latest
+registry.ci.openshift.org/origin/4.10:deploy quay.io/openshift/origin-deploy:4.10 quay.io/openshift/origin-deploy:4.10.0 quay.io/openshift/origin-deploy:latest
```
Comment (wking) [inline core-services/image-mirroring/openshift/mapping_origin_4_10:96]: hmm, we may want to pick more Cincy-scoped names for these images.

File: ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml [hunk 2]
```diff
@@ -56,6 +56,9 @@ images:
       as:
       - registry.access.redhat.com/ubi8/ubi:latest
   to: e2e-test
+promotion:
```
Comment (vrutkovs) [inline ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml:59]: We should either use
```
  additional_images:
    deploy: cincinnati
```
https://docs.ci.openshift.org/docs/getting-started/examples/#publishing-an-image-for-reuse

or simply `s/deploy/cincinnati/g` here

File: core-services/image-mirroring/openshift/mapping_origin_4_10 (hunk 1 above)
Comment (wking) [reply core-services/image-mirroring/openshift/mapping_origin_4_10]: I've added `excluded_images` with a2cc988039 -> e7b890bfc1, so now this doesn't add any mirroring entries.

File: ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml (hunk 2 above)
Comment (wking) [reply ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml:59]: I've added `excluded_images` with a2cc988039 -> e7b890bfc1, so now this doesn't add any mirroring entries.

**Review by LalatenduMohanty — APPROVED** [review]
/lgtm

**Pull request merged**